src/app/zeko/da-layer/da_utils/dune

```diff
@@ -0,0 +1,75 @@
+; This dune file is used to build the Rust library that is used by the OCaml
```

Comment:
Have you gotten any of this to work in the Nix world?

Reply:
If not, couldn't you just add this code to the Rust library already included in Mina?

Reply:
I haven't tried yet. I could but I wanted to make it future proof, when we will want to separate from the mina codebase.

Reply:
Yes, but compiling this together is a PITA.

Reply:
I’m not sure this is even a technically optimal solution. We will be duplicating all the Rust Mina libraries and have them in memory twice. I imagine this can go wrong easily.

Reply:
I would add this library inside of them and we would have it only once. But I would rather keep it this way if it works with nix.

Reply:
Why though? You can add your crate as a dependency and just re-export the interface I think.

Reply:
yeah that's possible as well